### PR TITLE
Implementation for add more elements to the list

### PR DESCRIPTION
**Pull request by AdriOli134**
- Add the restore filters button for reset the filters
- Add functionality to the Path field in the Examine panel, for display the folder path
- Add the Add button for add all the fields that are in the Path field.
- Add the MediaList for save the original list and when the user restore the filters it is possible to restore this original list
- Add remove button for each file in the play list, for remove it for the list

### CODE REVIEW

**Review by rodri-98 — APPROVED** [review]
Good job !

File: app/src/main/java/ju/video/player/controller/componentscontrollers/ButtonController.java [hunk 1]
```diff
@@ -55,129 +57,41 @@ public ButtonController(VideoListPanel playListPanel) {
...
-            return true;
-        }
-        return false;
-    }
-
-    public Path getCompletePath(int index) {
-        File completePath = new File(simplePathFilename.get(index));
-        Path path = Paths.get(pathOfTheSelectedFolder + "\\" + completePath);
-        return path;
+        examinePanel.setFieldText(pathOfTheSelectedFolder);
```
Comment (MariaAT18) [inline app/src/main/java/ju/video/player/controller/componentscontrollers/ButtonController.java:65]: Remove the comments if they are not necessary

File: app/src/main/java/ju/video/player/controller/componentscontrollers/ButtonController.java [hunk 2]
```diff
@@ -55,129 +57,41 @@ public ButtonController(VideoListPanel playListPanel) {
     @Override
     public void actionPerformed(ActionEvent e) {
         // closeFrame();
```
Comment (MariaAT18) [inline app/src/main/java/ju/video/player/controller/componentscontrollers/ButtonController.java]: Remove comments.

File: app/src/main/java/ju/video/player/controller/componentscontrollers/PlayMediaButtonController.java [hunk 3]
```diff
@@ -40,7 +40,7 @@ public void createList() {
         int listSize = ListValidVideos.getInstance().getVideoList().size();
         for (int position = 0; position < listSize; position++) {
             int index = (initialMedia + position) < listSize ? initialMedia + position: (initialMedia + position) - listSize;
-            String path = ListValidVideos.getInstance().getPathOfTheSelectedFolder() + "\\" + ListValidVideos.getInstance().getVideoList().get(index);
+            String path = ListValidVideos.getInstance().getVideoList().get(index).getAbsolutePath();
             System.out.println(path);
```
Comment (MariaAT18) [inline app/src/main/java/ju/video/player/controller/componentscontrollers/PlayMediaButtonController.java]: remove the System.out.println

**Review by MariaAT18 — CHANGES_REQUESTED** [review]

**Review by MariaAT18 — APPROVED** [review]

**Pull request merged**